distmv/uniform.go

```diff
@@ -56,7 +86,10 @@ func (u *Uniform) LogProb(x []float64) float64 {
 		panic(badSizeMismatch)
 	}
 	var logprob float64
-	for _, b := range u.bounds {
+	for i, b := range u.bounds {
+		if x[i] < b.Min || x[i] > b.Max {
+			logprob -= math.Inf(1)
```

Comment:
There is something strange here. Subtract infinity? Then subtract `math.Log(b.Max - b.Min)`?